filebeat/tests/system/test_registrar.py

```diff
@@ -1506,3 +1506,58 @@ def test_registrar_files_with_input_level_processors(self):
                 "inode": stat.st_ino,
                 "device": stat.st_dev,
             }, file_state_os)
+
+    def test_registrar_meta(self):
+        """
+        Check that multiple entries for the same file are on the registry when they have
+        different meta
+        """
+
+        self.render_config_template(
+            type='docker',
+            input_raw='''
```

Comment:
btw. check out the `textwrap` package. This allows you to indent multiline strings within the code more properly.